README.md

```diff
@@ -32,3 +52,5 @@ Run the ./refine file
 ```
 ./refine
 ```
+
+You can find more information on the [FAIRifier wiki](https://github.com/DTL-FAIRData/FAIRifier/wiki).
```

Comment:
I think Rajaram meant to add specific links to specific pages on the wiki. Makes more sense to me as well. Maybe you can update the main wiki page as well, right now it looks like it's an older version of the readme?